node/src/service.rs

```diff
@@ -328,6 +328,7 @@ pub fn new_partial(
 		client.clone(),
 	);
 
+	#[allow(clippy::redundant_clone)]
 	let (grandpa_block_import, grandpa_link) = sc_consensus_grandpa::block_import(
 		client.clone(),
```

Comment:
Can we not remove the clone and borrow?

Reply:
can try it as a separate activity. might have some additional dependencies